### PR TITLE
[WIP] makes the BSA more usable + bluespace designator

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
tweak: the bsa no longer makes a beam of death in front of it
add: bluespace designator, it can be used to mark land zones for express console or for the BSA
/:cl:

- [ ] find sprites

- [ ]  find sounds

- [ ]  fix code

- [ ] merge conflicts

bsa was never used because it literally destroys your station for no reason (why shoot a random laser???) and also you had to send mc tider with a gps as mobile target to make it work.
this makes it cool to use and improves rp by having people help each other blowing stuff up
the designator can be used to send supply rockets cause thats cool

**WORKS AS PROOF OF CONCEPT**

### CODE REVIEW

**Comment by FloranOtten** [comment]
There goes my cargo traitor strategy of setting up a BSA aiming toward Bridge

**Comment by necromanceranne** [comment]
What like the Archimedes II orbital cannon?

Also how long does the designator last for the express console? One shot or...?

**Comment by actioninja** [comment]
Lame. Just build it on the edge of the station.

File: code/game/objects/items/twohanded.dm [hunk 1]
```diff
@@ -869,3 +869,58 @@
 		C.change_view(CONFIG_GET(string/default_view))
 		user.client.pixel_x = 0
 		user.client.pixel_y = 0
+
+/obj/item/twohanded/binoculars/bsa
+	name = "Bluespace Designator"
+	desc = "This device creates a bluespace marker, which can be used as a gps or as an drop zone for pods."
```
Comment (XDTM) [inline code/game/objects/items/twohanded.dm]: an drop zone

File: code/modules/research/designs/electronics_designs.dm [hunk 2]
```diff
@@ -109,3 +109,13 @@
 	build_path = /obj/item/disk/nanite_program
 	category = list("Electronics")
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/nanite_disk
+	name = "Bluespace Designator"
+	desc = "It does stuff"
```
Comment (XDTM) [inline code/modules/research/designs/electronics_designs.dm]: /datum/design/nanite_disk

**Comment by habader** [comment]
what does any of this meeeeean????????

**Comment by zxaber** [comment]
It _would_ be kinda cool if we changed the sprite to have a clear tube showing off a death laser blasting into a portal.

**Comment by francinum** [comment]
There's no problem with the BSA's beam of death. If you are incapable of realizing that a large artillery piece is probably going to break what you put in front of it then that's your own fault.

**Comment by necromanceranne** [comment]
I feel like this is another pr that just sorta makes something considerably dumber for really not a whole lot of reason other than slightly veiled powergaming on the coders part. Part of the consideration when it comes to it's construction is making a reasonably large enough area to house it and also power it.

Tlal wants to slap it into cargo and use his designator to more easily bombard shit without any of the effort behind actually housing the gun.

**Comment by dr-bee** [comment]
Having a way to laser designate locations for someone to manually fire at would be nice, you can do it now with GPSes but this would at least prevent the gunner from accidentally murdering their spotter by firing before they throw the GPS.

If it still requires 2 people it is just an easier way to do what can already be done so there really isnt much of a balance issue.

**Comment by Tlaltecuhtli** [comment]
it requires:
roll the bsa as station objective
live long enough to have t4 parts
20 bluespace crystals
2 people with access to auth it
2 people to make it actually work

if it costs more than any other late game thing it should be at least able to shoot whatever you want


the laser doesnt make sense: if you are shooting something on station why does the laser go off map? why does the artillery piece have to be put in space because its impossible to keep it without losing all the air after 1 shot?

**Comment by Dennok** [comment]
On the Ship Battle site Gate away mission you can see that it actually installed in the vacuum.

**Comment by actioninja** [comment]
I've literally built a BSA platform with a pressurized control area it's not that hard.

**Comment by ninjanomnom** [comment]
Pull requests that are not at least ready to be tested are no longer allowed. Discuss the concept with others if you want feedback on the concept. Feel free to contact a maintainer about reopening this when it's more complete.

**Pull request closed without merging**